src/components/layout.css

```diff
@@ -584,3 +584,28 @@ details,
   height: 40px;
   bottom: 0;
 }
+
```

Comment:
refactor this pls to emotion

Reply:
Dang it, I had no idea we were using Emotion. I was going to ask for its inclusion.